<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCodePage</class>
 <widget class="QWidget" name="QCodePage">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>574</width>
    <height>139</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="useless_layout" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btn_open">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>110</horstretch>
            <verstretch>35</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Rubik</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(21, 21, 21);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}
QPushButton:hover{
	color: rgb(97, 186, 100);
}</string>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_run">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>110</horstretch>
            <verstretch>35</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Rubik</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(21, 21, 21);
	color: rgb(255, 255, 255);
	border-radius: 10px;
}
QPushButton:hover{
	color: #6a8ac8;;
}</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_path">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>35</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(21, 21, 21);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="leftMargin">
            <number>15</number>
           </property>
           <property name="topMargin">
            <number>7</number>
           </property>
           <property name="rightMargin">
            <number>15</number>
           </property>
           <property name="bottomMargin">
            <number>7</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineedit_path">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Rubik</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(97, 186, 100);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_codeedit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>35</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(21, 21, 21);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="box_codeedit">
           <property name="leftMargin">
            <number>16</number>
           </property>
           <property name="topMargin">
            <number>16</number>
           </property>
           <property name="rightMargin">
            <number>16</number>
           </property>
           <property name="bottomMargin">
            <number>16</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
